backend/src/main/java/io/debezium/configserver/rest/ConnectorResource.java

```diff
@@ -53,6 +55,7 @@
     public static final String FILTERS_VALIDATION_ENDPOINT = "/connector-types/{id}/validation/filters";
```

Comment:
Not directly related, but eventually, these endpoint should also be cluster-dependent (there could be different DBZs be installed). Can you log an issue for that?

Reply:
created #140